admin/sql/updates/schema-change/20.mirror.sql

```diff
@@ -10,8 +10,10 @@
 -- 20140318-series.sql
 -- 20140418-series-instrument-functions.sql
 -- 20140429-area-view.sql
-\set ON_ERROR_STOP 1
+\\set ON_ERROR_STOP 1
```

Comment:
Huh, how come extra slashes are being added here all of a sudden?  That's not valid syntax.

Reply:
looks like 12104a19eccae34b66910b36ecb8c723be92252e changed `<<EOSQL` to `<<'EOSQL'` which broke the escape sequence.  should be an easy fix.